README.md

```diff
@@ -23,14 +23,12 @@ These instructions will get you a copy of the role for your Ansible Playbook. On
 
 ### Prerequisities
 
-Ansible 2.4.3.0 version installed.
-Molecule 1.25 version installed.
-Ansible-Lint 3.4.20 version installed.
-python-consul 1.1.0 version installed
+Ansible >= 2.4.3.0
```

Comment:
is not consistent with version defined in test-requirements.txt file

Reply:
Updated